packages/web-components/src/components/va-link-action/va-link-action.tsx

```diff
@@ -0,0 +1,115 @@
+import { Component, Event, EventEmitter, Host, h, Prop, State, Watch } from '@stencil/core';
+import classNames from 'classnames';
+
+/**
+ * @componentName Link - Action
+ * @maturityCategory caution
+ * @maturityLevel candidate
+ */
+
+@Component({
+  tag: 'va-link-action',
+  styleUrl: 'va-link-action.css',
+  shadow: true,
+})
+export class VaLinkAction {
+  /**
+   * If `true`, the component-library-analytics event is disabled.
+   */
+  @Prop() disableAnalytics?: boolean = false;
+
+  /**
+   * The href attribute of the anchor.
+   */
+  @Prop() href!: string;
+
+  /**
+   * The anchor text.
+   */
+  @Prop() text!: string;
+
+  /**
+   * An optional message that will be read by screen readers when the link is focused.
+   */
+  @Prop() ariaDescribedBy?: string;
```

Comment:
I have had problems with props that are named with `aria-` before. a11y tests will fail in vets-website.

I think for this we would need to do similar to other components that have `message-aria-describedby` prop.  [va-telephone as an example](https://github.com/department-of-veterans-affairs/component-library/pull/970/files).

Reply:
Thank you for pointing that out. I initially named the props with `message-aria-`, but I have changed them back. I wasn't aware that this could cause accessibility tests to fail on the vets-website. I'll follow the approach used in other components. Thanks again for your feedback!

Reply:
@harshil1793 No problem! [This is the error I had before](https://github.com/department-of-veterans-affairs/component-library/pull/620) from the axe check. I think if you added that prop in [your axe test](https://github.com/department-of-veterans-affairs/component-library/blob/06de34d660630e9f5c23c80786d4df50d3bec88a/packages/web-components/src/components/va-link-action/test/va-link-action.e2e.ts#L62-L68), then you would see the same thing.